pkg/utils/gpu.go

```diff
@@ -143,12 +143,20 @@ func IsMarinerEnabledGPUSKU(vmSize string) bool {
 // NVv3 is untested on AKS, NVv4 is AMD so n/a, and NVv2 no longer seems to exist (?).
 func GetGPUDriverVersion(size string) string {
 	if UseGridDrivers(size) {
-		return Nvidia535GridDriverVersion
+		return NvidiaGridDriverVersion
 	}
 	if isStandardNCv1(size) {
 		return Nvidia470CudaDriverVersion
```

Comment:
The 470 driver is not being updated as it is only required for this NCv1 size, which is not supported and has not ben supported for a while. I don't think there are any karpenter/NAP users of it.